src/Traits/AuthoredNotesTrait.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Arcanedev\LaravelNotes\Traits;
+
+use Arcanedev\LaravelNotes\Models\Note;
+use Illuminate\Support\Facades\Config;
+
+/**
+ * Trait     AuthoredNotesTrait
+ *
+ * @package  Arcanedev\LaravelNotes\Traits
+ * @author   ARCANEDEV <arcanedev.maroc@gmail.com>
+ */
+trait AuthoredNotesTrait
+{
+    /* -----------------------------------------------------------------
+     |  Relationships
+     | -----------------------------------------------------------------
+     */
+
+    /**
+     * Relation to ONE note.
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\MorphOne
+     */
+    public function authoredNotes()
```

Comment:
You can rename it to `notes`  instead of `authoredNotes`, example:

```php
$notes = $user->notes()->latest()->get();
```

Reply:
no, because this might conflict with the `HasManyNotes` Trait! For example, if you want to add `Notes` to a `User` object, then the `User` would need both Traits! This would result in a conflict!